### DISCUSSION TITLE
The automation has an action that calls an unknown service

### DISCUSSION THREAD

**Discussion by fionnanburke** [Blackymas/NSPanel_HA_Blueprint#1828] [Q&A]
I have tried to upgrade my NSPanels to the latest release this week and it didn't go well! 
I'm focusing on one at the moment and after much trial and error I was able to get the TFT installed however, the blueprint is causing errors to appear and no changes are being made to the panel. 

What I think caused this issue is that at some point in TFT upload attempts I renamed the device from "nspanelfbstudy" to "Fionnans Study Panel". This caused issues so I reverted the change and the panel in HA and ESPHome is now back to "nspanelfbstudy" but it appears that the old name is cached somewhere. 

When I create a new blueprint and select "nspanelfbstudy" as the device and make no other changes / configs to the blueprint I get these errors in HA:

```
`FBStudy - NSPanel Config uses an unknown service
The automation "FBStudy - NSPanel Config" (automation.fbstudy_nspanel_config) has an action that calls an unknown service: esphome.fionnans_study_panel_page_home.

This error prevents the automation from running correctly. Maybe this service is no longer available, or perhaps a typo caused it.

To fix this error, [edit the automation](http://192.168.2.48:8123/config/automation/edit/1708619182523) and remove the action that calls this service.

Click on SUBMIT below to confirm you have fixed this automation.`

```
In Developer Tools, there are no services "esphome.fionnans_study_panel", only for "esphome.nspanelfbstudy"

I have removed the device from the ESPHome integration, made sure all entities are deleted and rebooted HA but when I add the device back it will always come in as "Fionnans Study Panel". 

As a result of these issues, the NSPanel can't be configured, I can't get anything to show on the screen except the time and this little red HA icon to the right. 
![nspanel](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/19493718/4a49f67f-9ed2-4af7-9297-ad1905e421ad)

And this is what I see in HA:
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/19493718/2806b269-795a-4ebb-87c0-75177dd7f0ba)

My ESP Config:

```substitutions:
  # Settings - Editable values
  device_name: "nspanelfbstudy"
  wifi_ssid: "----"
  wifi_password: "----"
  #nextion_update_url: "http://192.168.2.48:8123/local/nspanel_eu.tft"  # Optional
  nextion_update_url: "http://192.168.1.199/nspanel_eu.tft"
  # Add-on configuration (if needed)
  # heater_relay: "1"  # Possible values: "1" or "2"

# Customization area
##### My customization - Start #####
##### My customization - End #####

# Core and optional configurations
packages:
  remote_package:
    url: https://github.com/Blackymas/NSPanel_HA_Blueprint
    ref: main
    files:
      - nspanel_esphome.yaml # Core package
      # Optional advanced and add-on configurations
      # - advanced/esphome/nspanel_esphome_advanced.yaml
      # - nspanel_esphome_addon_climate_cool.yaml
      # - nspanel_esphome_addon_climate_heat.yaml
      # - nspanel_esphome_addon_climate_dual.yaml
    refresh: 300s

esp32:
  framework:
    type: esp-idf

##### LOGGER #####
#logger:
#  baud_rate: 0
#  level: WARN

wifi:
  networks:
    - manual_ip:
        static_ip: 192.168.2.225
        gateway: 192.168.2.0
        subnet: 255.255.255.0
  power_save_mode: none ```


Has anybody seen this before? Any advice?

**Comment by edwardtfn** [comment]
- #1830
